### ISSUE TITLE
Prod deployment issue

### ISSUE THREAD

**Issue by ipzol** [damienbod/AngularWebpackVisualStudio#135]
Hi, thank you for all work you've done, it is terrific. I followed earlier versions with angular 4 + AOT, everything worked fine. With the most recent changes it does not work when deployed to IIS, it stuck at point 'Loading...'. I did all standard steps as before. Created not manged pool, virtual directory, web application. Something is missed, the angular part is not bootstrapping or nor routed to. I use IIS server on local Windows 10 box. Everything works from  Visual Studio. Could you please advise?

**Comment by damienbod** [comment]
Hi @ipzol 

When you do the production build 

npm install
npm run build-production

Do you get any warnings?

Greetings Damien

**Comment by ipzol** [comment]
Hi,

I took project from repo. For production build and publish no warnings or errors. The return code is 0. The npm  install warned about typescript version mismatched, which is not an issue. 

I started with my project and faced that I can’t run it on IIS, decided to try project from repo and deployed it to IIS, same issue.
It is not Angular 5 itself, other Angular 5 project .Net MVC (not Core) published from VS works fine. I copied everything from Release\netcoreapp2.0\publish to target folder.
It might be that some plumbing is missed, IIS can’t find Angular stuff.

Are you able to deploy and run project from local IIS (not IIS express)? 

Thanks,
Igor.


Sent from Mail for Windows 10
